addons/project/models/project_project.py

```diff
@@ -801,8 +801,8 @@ def _get_stat_buttons(self):
                 icon = 'frown-o text-danger'
             buttons.append({
                 'icon': icon,
-                'text': _lt('Satisfaction'),
-                'number': f'{round(100 * self.rating_avg_percentage, 2)} %',
+                'text': _lt('Average Rating'),
+                'number': f'{round(self.rating_avg, 1) if self.rating_avg % 1 != 0 else int(self.rating_avg)} / 5',
```

Comment:
```suggestion
                'number': f'{int(self.rating_avg) if self.rating_avg.is_integer() else round(self.rating_avg, 1)} / 5',
```
As you wish, but I find this slightly more explicit